services/src/main/java/org/keycloak/services/managers/ApplianceBootstrap.java

```diff
@@ -127,4 +141,31 @@ public void createMasterRealmUser(String username, String password) {
         ServicesLogger.LOGGER.addUserSuccess(username, Config.getAdminRealm());
     }
 
+    public void createTemporaryMasterRealmAdminService(String clientId, String clientSecret /*, Integer expriationMinutes*/) {
+        RealmModel realm = session.realms().getRealmByName(Config.getAdminRealm());
+        session.getContext().setRealm(realm);
+
+        clientId = StringUtil.isBlank(clientId) ? DEFAULT_TEMP_ADMIN_SERVICE : clientId;
+        //expriationMinutes = expriationMinutes == null ? DEFAULT_TEMP_ADMIN_EXPIRATION : expriationMinutes;
+
+        ClientModel adminClient = session.clients().addClient(realm, clientId);
```

Comment:
Looks like the rebase also rolled back the changes from my PR.

Reply:
Ugh, I just need to take the day off at this point :(